From c2794f13a27ed31c202fe182c8360903dc3849d3 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 4 Apr 2019 15:45:15 +0800
Subject: [PATCH 04/30] gpio: gpio-mxc: fix compile error

Fix compile error which caused by modification in the wrong line.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpio/gpio-mxc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/gpio-mxc.c b/drivers/gpio/gpio-mxc.c
index 8c1fad28dfef..baeab10f49ec 100644
--- a/drivers/gpio/gpio-mxc.c
+++ b/drivers/gpio/gpio-mxc.c
@@ -766,8 +766,6 @@ static int __maybe_unused mxc_gpio_noirq_suspend(struct device *dev)
 	unsigned long flags;
 	int ret;
 #ifdef CONFIG_GPIO_MXC_PAD_WAKEUP
-	int wakeup_line = mxc_gpio_get_pad_wakeup(port);
-
 	mxc_gpio_set_pad_wakeup(port, false);
 #endif
 
@@ -799,6 +797,8 @@ static int __maybe_unused mxc_gpio_noirq_resume(struct device *dev)
 	int ret;
 
 #ifdef CONFIG_GPIO_MXC_PAD_WAKEUP
+	int wakeup_line = mxc_gpio_get_pad_wakeup(port);
+
 	mxc_gpio_set_pad_wakeup(port, true);
 #endif
 	if (mxc_gpio_hwtype == IMX21_GPIO)
-- 
2.17.1

